schemas/dab.draft.schema.json

```diff
@@ -1,16 +1,20 @@
 {
   "$schema": "https://json-schema.org/draft-07/schema",
-  "$id": "#/additionalProperties/version",
+  "$id": "https://github.com/Azure/data-api-builder/releases/download/vmajor.minor.patch/dab.draft.schema.json",
   "title": "Data API builder",
   "description": "Schema for Data API builder engine",
-  "additionalProperties": {
-    "version": "https://github.com/Azure/data-api-builder/releases/download/vmajor.minor.patch/dab.draft.schema.json"
-  },
+  "additionalProperties": false,
   "type": "object",
   "properties": {
+    "$schema": {
+      "type": "string",
+      "description": "URI of the JSON schema that can validates the structure, constraints, and data types of this Data API builder runtime configuration file.",
```

Comment:
```suggestion
      "description": "URI of the JSON schema with which the structure, constraints, and data types of this Data API builder runtime configuration file should conform to.",
```

Reply:
updated suggestion to not have 'to.' at the end. unneeded.